src/videojs.ima.js

```diff
@@ -305,7 +305,13 @@
         player.ads.startLinearAdMode();
       }
       adContainerDiv.style.display = 'block';
-      controlsDiv.style.display = 'block';
+      // Don't show ad controls for not video ads (like modal ads)
+      if (adEvent.getAd().getContentType().search(/video/i) !== 0) {
```

Comment:
I'm not sure that this is quite right - if "video" is in the string, it will return a number >=0, and we want to show controls. Otherwise it will return -1, and we do not want to show the controls. So this should be 

if (adEvent.getAd().getContentType().search(/video/i) >= 0) {
  controlsDiv.style.display = 'block';
} else {
  controlsDiv.style.display = 'none';
}